docs/install/knative-with-operators.md

```diff
@@ -0,0 +1,238 @@
+---
+title: "Installing Knative with Knative Operators"
+weight: 10
+type: "docs"
+---
+
+# Knative installations with Knative Operators
+
+As of v0.10.0, Knative started to release Knative operators as tools to install, configure and manage Knative. This
+guide will explain how to install and uninstall Knative with Knative operators.
+
+There are two major components in Knative: Serving and Eventing. There are two operators: [Knative Serving Operator](https://github.com/knative/serving-operator)
+and [Knative Eventing Operator](https://github.com/knative/eventing-operator), respectively dedicated to each of them. Please make sure you have set up a Kubernetes
+cluster accessible to your local workstation.
+
+## Limitations of Knative Operators:
+
+Knative operators are still in Alpha phase. They only focus on Knative installation for the generic Kubernetes platform,
+which means Docker-Desktop Kubernetes on Mac and Minikube can be directly used as the Kubernetes clusters for you to install
+Knative with Knative operators, without any additional configuration out of the scope of the operator custom resource,
+but vendor-specific Kubernetes services, like Google Kubernetes Engine, IBM Kubernetes Service, OpenShift, etc, may need
```

Comment:
Make this a new sentence for better readability?
e.g.
"Vendor specific...[]"

change like -> such as? Maybe remove the "etc"?
e.g.
"...such as GKE, IBM Kubernetes Service or Red Hat OpenShift"

Reply:
Yes, this is a really long sentence. Maybe split it as:

> The focus has been on installation on a generic Kubernetes platform (such as Docker Desktop, Minikube, or kubeadm clusters), and the operator does not perform any platform specific customization.

I don't think we're suggesting that the operator won't work on GKE, IKS, or OpenShift, just that it won't be integrated and you might to do some additional work.